src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/ClaimController.java

```diff
@@ -119,19 +118,12 @@ public Claim requestMoreTimeToRespond(@PathVariable("externalId") String externa
     @GetMapping("/{caseReference}/defendant-link-status")
     @ApiOperation("Check whether a claim is linked to a defendant")
     public DefendantLinkStatus isDefendantLinked(@PathVariable("caseReference") String caseReference) {
-        Boolean linked = claimService.getClaimByReferenceAnonymous(caseReference)
+        boolean linked = claimService.getClaimByReferenceAnonymous(caseReference)
             .filter(claim -> claim.getDefendantId() != null)
             .isPresent();
         return new DefendantLinkStatus(linked);
     }
 
-    @PostMapping(value = "/{externalId:" + UUID_PATTERN + "}/pre-payment")
```

Comment:
Please make sure to remove call to this endpoint in `citizen-frontend` first and merge that in master before merging this PR.
This is the code that needs to be removed in frontend:
https://github.com/hmcts/cmc-citizen-frontend/blob/master/src/main/app/claims/claimStoreClient.ts#L40

Reply:
Hi @sabahirfan. The https://github.com/hmcts/cmc-citizen-frontend/pull/1430/files has the change already. Will ask Paul to test it first, then merge the frontend first and then backend. Thats the plan already.

Reply:
Thanks